design/ip-address-manager/ip-address-management-for-networkdata.md

```diff
@@ -169,13 +169,17 @@ spec:
       subnet: 192.168.0.0/24
       gateway: 192.168.0.1
       prefix: 24
+      dnsServers:
+        - 8.8.8.8
```

Comment:
Not related to this PR but I notice the prefix is duplicated since the subnet is specified as a CIDR - is that intentional?

Reply:
yes, it is for two different uses. the CIDR is used to give the boundaries of the allocation addresses, while prefix is used for the configuration. Maybe an example could make this clearer : Let's say we have a /24 on that network. However, we want the IP addresses to be allocated from the first /25 only, then the CIDR will be /25, but the prefix given will be /24 so that the host can reach the whole subnet. Does this make sense @hardys  ?